pkg/api/validation/validation.go

```diff
@@ -38,6 +38,10 @@ import (
 	"github.com/golang/glog"
 )
 
+// TODO: delete this global variable when we enable the validation of common
+// fields by default.
+var FixUpdateAtServer bool = true
```

Comment:
Done. Sorry I was too lazy to fix this as I thought the variable will be short lived.